docs/socialmedia.md

```diff
@@ -6,4 +6,8 @@ DietPi announces are done via Twitter: <https://twitter.com/DietPi_>.
 
 ## DietPi on Facebook
 
-DietPi announces and discussions are done via Facebook: <https://www.facebook.com/groups/167403007229675>
+DietPi announces and discussions are done via Facebook: <https://www.facebook.com/groups/167403007229675>.
```

Comment:
I think after the colon, a dot is not required anymore to end the sentence, is it?

Reply:
Ok. Changed for all entries.